azkaban-common/src/main/java/azkaban/executor/ExecutableFlow.java

```diff
@@ -50,11 +50,13 @@
   public static final String AZKABANFLOWVERSION_PARAM = "azkabanFlowVersion";
   public static final String IS_LOCKED_PARAM = "isLocked";
   public static final String FLOW_LOCK_ERROR_MESSAGE_PARAM = "flowLockErrorMessage";
+  public static final String EXECUTION_SOURCE = "execution_source";
```

Comment:
Could you please use `camelCase` in the property name to follow the naming convention used in the rest of the properties?

Reply:
yes, thanks for pointing